ui/app/pages/permissions-connect/index.scss

```diff
@@ -1,25 +1,36 @@
-@import 'permissions-connect-footer/index';
-
 @import 'choose-account/index';
 
 .permissions-connect {
   width: 100%;
   position: relative;
   background: white;
 
-  &__page-count-wrapper {
-    margin-top: 26px;
-    margin-left: 20px;
+  &__top-bar--two-items,
```

Comment:
Nit: I think these could be consolidated into one class by using CSS grid instead of flexbox here. CSS grid would let you declare two columns, and place the page-count in the second column regardless of whether there is one or two elements.

e.g.:
in the container:
```
display: grid;
grid-template-columns: 1fr 1fr;
```
in `__page-count`:
```
grid-column: 2;
justify-self: end;
```

Reply:
Suggested changes applied in cdd4c6125